Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,16 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import HeaderTitle from "../HeaderComponents/HeaderTitle";
+import "./Header.css";
+
+const HeaderContent = () => (
+  <div class="header">
+    <HeaderTitle />
```

Comment:
I think the idea w/ this project is passing props. So you would write line 7 like `<HeaderTitle h3Property="Lambda School" />`. Then access it like I'll show in the HeaderTitle comment.